pyvista/plotting/plotter.py

```diff
@@ -3728,7 +3728,7 @@ def add_mesh(
         prop = Property(**prop_kwargs)
         actor.SetProperty(prop)
 
-        if style == 'points_gaussian':
+        if style == 'points_gaussian' and self.mapper.dataset is not None:
```

Comment:
`self.mapper.dataset` type is `DataSet | None`.  This is probably the right fix from a typing standpoint.  It is _possible_ that it should be only `DataSet` here and a different fix is needed, but given the complexity of `add_mesh` I'm not confident that it is true.

Reply:
If it is `None`, then we just stop setting the parameters for `point_gaussian` which seems like an okay result to me.

Reply:
Yup, I'm not at all aware of all the mechanics here, but the fix seems to err on the safe side.